certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -333,11 +337,23 @@ DEFINE
  TARGETSHARETYPE           = NUMBER
  TARGETIRSCODE             = NUMBER
  TMPVAL                    = NUMBER
- END [DEFINE]
+
+[* TKainz - added 03/14/24
+*]
+ PARAMXFER3HANDLING        = NUMBER
```

Comment:
I think this should be grouped with all the other PARAM variables.

Reply:
Done